report_dynamic/demo/demo.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <!-- template for demo_report_1 -->
+    <record id="demo_template_1" model="report.dynamic">
```

Comment:
Don't know it is my problem only, but noupdate flag is ignored and secont test fire up is blocked by _constrain_template_status